pkg/operator/ceph/object/controller.go

```diff
@@ -483,3 +490,24 @@ func (r *ReconcileCephObjectStore) startMonitoring(objectstore *cephv1.CephObjec
 	logger.Info("starting rgw healthcheck")
 	go rgwChecker.checkObjectStore(r.objectStoreChannels[objectstore.Name].stopChan)
 }
+
+func (r *ReconcileCephObjectStore) verifyObjectUserCleanup(objectstore *cephv1.CephObjectStore) (reconcile.Result, bool) {
```

Comment:
please add a unit test for this method. It should be fairly straight forward to create the mocked object users and confirm if the return is true or false based on whether there are users.

Reply:
Done